### PR TITLE
KINNO-647-remove_acount_info_actitvity

### PR DESCRIPTION
**Pull request by shayb**
#### Main purpose:
remove old account info 
#### Technical description:
use the module new look and features
#### Dilemmas you faced with? 
(Not a must)
#### Tests:
(Main tests you added/changed)
#### Documentation:
(Source/readme.md)

### CODE REVIEW

File: sdk/src/main/java/com/kin/ecosystem/transfer/view/AccountInfoActivity.java [hunk 1]
```diff
@@ -1,93 +1,44 @@
...
+            if (authToken != null && authToken.isActivated()) {
+                return BlockchainSourceImpl.getInstance().getPublicAddress();
+            } else {
+                throwExceptionOnDataError();
+            }
+        } catch (BlockchainException e) {
+            throwExceptionOnDataError();
         }
-        super.onBackPressed();
+        return "";
```
Comment (OrenZak) [inline sdk/src/main/java/com/kin/ecosystem/transfer/view/AccountInfoActivity.java]: why not null or throw an exception?

File: core/src/main/java/com/kin/ecosystem/core/data/auth/AuthRepository.java [hunk 2]
```diff
@@ -145,6 +145,13 @@ public AuthToken getAuthTokenSync() {
 		}
 	}
 
+    @Override
+    public boolean isCachedAuthTokenExpired() {
+        if (cachedAuthToken != null) {
+            return isAuthTokenExpired(cachedAuthToken);
```
Comment (OrenZak) [inline core/src/main/java/com/kin/ecosystem/core/data/auth/AuthRepository.java:151]: check also from `local`

File: sdk/src/main/java/com/kin/ecosystem/transfer/view/AccountInfoActivity.java [hunk 3]
```diff
@@ -1,93 +1,42 @@
...
-                if (presenter != null) {
-                    presenter.closeClicked();
-                }
-            }
-        });
+    public void updateTransactionInfo(String senderAppId, String senderAppName, String receiverAppId, String memo) {
+        super.updateTransactionInfo(senderAppId, senderAppName, receiverAppId, memo);
+        //server needs to check transaction got with that memo on the blockchain
+        //if found add senderAppName to transaction history
+        Log.d("AccountInfoActivity", "AccountInfoActivity Validate memo " + memo + " on blockChain and if valid add to transaction history sender App Name " + senderAppName + " sender app id: " + senderAppId + " receiver ppp Id: " + receiverAppId);
```
Comment (OrenZak) [inline sdk/src/main/java/com/kin/ecosystem/transfer/view/AccountInfoActivity.java]: Delete this Log.d and use our Log object.

File: sdk/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -46,6 +46,8 @@
     <string name="kinecosystem_transfer_title">In order to send Kin, %1$s would like to receive your Kin account information (i.e. your public address) from %2$s.</string>
 
     <string name="kinecosystem_kin_is_a_digital_currency_that_makes">Kin is a digital currency that makes\nyour favorite apps even better.</string>
+    <string name="kinecosystem_transfer_account_info_error_relogin_title"><![CDATA[You need to log out & back into %1$s before transferring Kin.]]></string>
+
```
Comment (OrenZak) [inline sdk/src/main/res/values/strings.xml:50]: what is `CDATA` ?

**Review by OrenZak — DISMISSED** [review]
See my comments

File: sdk/src/main/res/values/strings.xml (hunk 4 above)
Comment (shayb) [reply sdk/src/main/res/values/strings.xml:50]: its beacuse of the & sign - this is the string generated by the studio

**Review by OrenZak — APPROVED** [review]
💯

**Pull request merged**